packages/core/core.v0.12.2/opam

```diff
@@ -1,5 +1,5 @@
 opam-version: "2.0"
-version: "v0.12.1"
+version: "v0.12.2"
```

Comment:
```suggestion
```

Reply:
(sorry I wasn't able to commit your suggestion for some unknown reason) but thanks!!